tests/test_client.py

```diff
@@ -98,5 +103,44 @@ def test_failure_url_param(self):
         url = self.lp.get_buy_url(item, failure_url="http://example.com")
         self.assertTrue('failure_url' in url)
 
+    def test_get_web_url_product_key_param(self):
+        item = ItemDefinition(1, 'EUR20', 'DE19.0', 'http://help.me/', 'title')
```

Comment:
While I'm sure http://help.me/ is going to be a fascinating site, would you mind just using example.com please? ;)